add weak hashing algorithm entry (#31)
Ostorlab/KB: MOBILE_CLIENT/COMMON/_MEDIUM/WEAK_HASHING_ALGO/recommendation.md

> We recommend `SHA-256` to verify file signatures ans certificates.

Reviewer: ```suggestion
We recommend `SHA-256` to verify file signatures and certificates.
```

Reviewer: SHA 256 is not defined in the description.

> For passwords hashing we sugest using a slow and strong algorithm like `Argon2` or `Bcrypt`, it is best to avoid fast algorithms for this use.

Reviewer: ```suggestion
For password hashing we suggest using a dedicated strong algorithm like `Argon2` or `Bcrypt`.
```

Reviewer: You need to explain why those for that use case. Fast and Slow doesn't cut it.
